### PR TITLE
[SC-5730] A11y to account page

### PR DESCRIPTION
**Pull request by qnowak**
# Description

## Links to Tickets or other pull requests

- https://ticketsystem.schul-cloud.org/browse/SC-5730

## Changes
- Added a11y to account page 
- Changed pwd scripts
- Little change regarding to pwd on Login page

## Approval for review
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: views/account/settings.hbs [hunk 1]
```diff
@@ -1,203 +1,220 @@
...
+	{{#content "page"}}
+
+		{{message}}
+		{{error}}
+		<div class="route-account">
+			{{#unless sso}}
+				<form method="post">
+					{{#ifneq currentRole "Schüler"}}
+						<div class="form-group">
+							<label for="firstName">Vorname:</label>
```
Comment (luizzaly) [inline views/account/settings.hbs:19]: please use i18 here for the label: account.global.label.firstName

File: views/account/settings.hbs [hunk 2]
```diff
@@ -1,203 +1,220 @@
...
+							<input id="firstName" type="text" class="form-control" name="firstName"
+								value="{{ ../currentUser.firstName }}">
+						</div>
+
+						<div class="form-group">
+							<label for="lastName">Nachname:</label>
```
Comment (luizzaly) [inline views/account/settings.hbs:25]: same here $t "account.global.label.lastName :)

File: views/authentication/forms/login.hbs [hunk 3]
```diff
@@ -15,62 +15,76 @@
...
+						<button title={{$t "login.button.less"}}
+							class="btn toggle-btn btn-block btn-secondary btn-hide-providers"><i
+								class="fa fa-chevron-up"></i>{{$t "login.button.less"}}</button>
 					</div>
 				</div>
 				{{> "lib/components/csrfInput"}}
 				<div class="form-group">
 					{{#if notification.statusCode}}
-						<input type="submit" id="submit-login" class="btn btn-block btn-primary btn-login" value="{{$t "login.button.login"}}" data-testid="submit-login" data-timeout="{{notification.timeToWait}}" disabled/>
+					<input type="submit" id="submit-login" class="btn btn-block btn-primary btn-login"
```
Comment (haniapalac) [inline views/authentication/forms/login.hbs:82]: I guess we need to decide how to deal with disabled input fields/options etc.

**Comment by haniapalac** [comment]
the 'show password' button is still not tabbable + required fields are not announced (ChromeVox)
image

File: views/account/settings.hbs [hunk 4]
```diff
@@ -1,203 +1,220 @@
...
-					{{$t "account.text.centralIdentityManagement"}}
-                </p>
-            </div>
-            {{/unless}}
-
-            {{#unless (userHasRole "student")}}
+					<div>
+						<div class="form-group">
+							<label>Vorname</label>
+							<input type="text" class="form-control" value="{{ currentUser.firstName }}" readonly>
```
[the remark this answers is not in the document]
Comment (luizzaly) [reply views/account/settings.hbs:99]: I think its okay having it, as the screen reader reads the field out..

**Pull request merged**